lib/OpenQA/CacheService/Task/Sync.pm

```diff
@@ -39,10 +41,15 @@ sub _cache_tests {
     my @cmd = (qw(rsync -avHP --timeout), RSYNC_TIMEOUT, "$from/", qw(--delete), "$to/tests/");
     my $cmd = join ' ', @cmd;
     $ctx->info("Calling: $cmd");
-    my $output = `@cmd`;
-    my $status = $? >> 8;
+    my $status;
+    for my $retry (1 .. RSYNC_RETRIES) {
+        my $output = `@cmd`;
+        $status = $? >> 8;
+        $job->note(output => "[info] [#$job_id] Calling try $retry: $cmd\n$output");
```

Comment:
Is it intentional that the latest result overwrites the previous one? I think for other retry features we log all attempts.

Reply:
I guess it's ok either way

Reply:
no ,isn't , thx for spotting.